index.js

```diff
@@ -8,43 +8,65 @@ var hasFlag = function (flag) {
 	return pos !== -1 && (terminator !== -1 ? pos < terminator : true);
 };
 
-module.exports = (function () {
-	if ('FORCE_COLOR' in process.env) {
-		return true;
+var support = function (level) {
+	if (level === 0) {
+		return null;
```

Comment:
ping @sindresorhus / https://github.com/chalk/supports-color/pull/25#discussion-diff-33635307R11

Note to self, additive commits in PRs + squash before merging in the future.